lib/modules/datasource/crate/__snapshots__/index.spec.ts.snap

```diff
@@ -299,7 +299,7 @@ exports[`modules/datasource/crate/index > getReleases > processes real data: lib
       "version": "0.2.50",
     },
     {
-      "version": "0.2.51",
+      "version": "0.2.51+metadata",
```

Comment:
I believe the build metadata will eventually go through `applyDatasourceFilters`, so should be fine here.

(though I haven't found a place to test this)